CHANGELOG.md

```diff
@@ -1,5 +1,95 @@
 # DEICODE changelog
 
+## Version 0.1.7 (2019-4-2)
+
+Implemented in [PR#29](https://github.com/biocore/DEICODE/pull/29).
+
+### Features
+
+* The `--min-feature-count` and `--iterations` options are now usable when
+  running DEICODE outside of QIIME 2.
+
+* When running DEICODE outside of QIIME 2, it will no longer raise an error if
+  the specified output directory doesn't exist (it'll automatically try to
+  create the output directory, even creating multiple levels of directories if
+  needed).
+
+* Options' default values (if applicable) are now shown when running
+  `deicode --help`.
+
+* The `--in-biom` and `--output-dir` options are now marked as required when
+  running DEICODE outside of QIIME 2 (so just running, e.g., `deicode` without
+  any options will give you a clearer error message than before).
+
+* The RPCA functionality is now exposed via the `deicode.rpca` module,
+  which contains an `rpca()` function.
+
+### Backward-incompatible changes [stable]
+
+* The following option names have changed when running DEICODE outside of QIIME
+  2:
+
+| Original Name        | New Name             |
+| -------------------- | -------------------- |
+| `--in_biom`          | `--in-biom`          |
+| `--output_dir`       | `--output-dir`       |
+| `--min_sample_depth` | `--min-sample-count` |
+
+* `deicode.scripts._rpca` has been replaced by `deicode.scripts._standalone_rpca`.
+  * Similarly, the `rpca()` function within `deicode.scripts._rpca` has been
+    replaced by the `standalone_rpca()` function in
+    `deicode.scripts._standalone_rpca`.
+
+### Backward-incompatible changes [experimental]
+
+### Performance enhancements
+
+### Bug fixes
+
+* Some of the QIIME 2 RPCA behavior was not mirrored perfectly in the non-QIIME
+  2 RPCA code. So there are a few "new" things done by the non-Q2 RPCA code that
+  it didn't do before:
+  * Uses `--min-feature-count` with a default value of `10`. Previously, the
+    non-Q2 RPCA code didn't do this filtering step at all.
+  * Adds a PC3 containing zeros if the `rank` is set to `2`.
```

Comment:
@cameronmartino would probably know best! From the sounds of it (per his comment above), it sounds like this was because Emperor previously couldn't handle this (but it now can).

I actually removed this behavior from the RPCA code earlier today. Maybe it'd be a good idea (in the future) to add a test with `--rank 2` and seeing what happens when it's visualized with Emperor?

Reply:
I believe this broke Emperor before the 2D option was available, that is why it is in there. I could be mistaken though.